src/main/webapp/login.html

```diff
@@ -1,22 +1,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>altruism-ator</title>
+    <title>Login</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-      <header>
-        <div class="container">
-            <nav>
-                <ul>
-                    <li class="nav-item"><a href="index.html">Home</a></li>
-                    <li class="nav-item"><a href="profile.html">Profile</a></li>
-                    <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
-                    <li class="nav-item"><a href="chat.html">Chat</a></li>
-                </ul>
-            </nav>
-        </div>
-      </header>
+    <header>
+      <div class="container">
+        <nav>
+          <ul>
+            <li class="nav-item"><a href="index.html">Home</a></li>
+            <li class="nav-item"><a href="profile.html">Profile</a></li>
+            <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
+            <li class="nav-item"><a href="chat.html">Chat</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="loginForm">
+        <form action="/login" method="POST">
+          <label for="username">Username:</label>
+          <input type="text" name="username" required><br><br>
+          <label for="password">Password:</label>
+          <input type="password" name="password" required><br><br>
+          <input type="submit">
+        </form>
+        <a href="signUp.html">
+          <button>Sign Up</button>
```

Comment:
You don't need this button, since the input type submit on line 26 should suffice the action of this one.

Reply:
My thought is that on the login page, the user should have the option to go directly to the sign in page even without entering any login information (since they don't have one). But even with the submit button, it will be connected to the login servlet, and I don't plan to direct the user to the sign-up page even if unsuccessful login.

Reply:
I see what you mean, I like your thought process! keep it as you have it.